src/stylesheets/components/_forms.scss

```diff
@@ -1,144 +1,49 @@
-$usa-form-width: 32rem;
-
-[type=submit] {
-  @include media($medium-screen) {
-    margin-top: calc(3rem - .2em); // Label margin minus input bottom margin
-  }
-}
-
-fieldset {
-  border: none;
-  margin: 0;
-  padding: 0;
-}
-
 .usa-form {
-  @include media($medium-screen) {
-    max-width: $usa-form-width;
-  }
-}
-
-.usa-input-tiny {
-  @include media($medium-screen) {
-    max-width: 6rem;
-  }
-}
-
-.usa-input-medium {
-  @include media($medium-screen) {
-    max-width: 12rem;
-  }
-}
-
-.usa-form-note {
-  float: right;
-  font-family: $font-sans;
-  font-size: $h5-font-size;
-  margin: 0 0 1.5rem;
-
-  + * {
-    clear: both;
-  }
-}
-
-.usa-form-large {
-  @include media($medium-screen) {
-    max-width: 46rem;
-  }
-}
-
-.usa-input-grid {
-  @include media($medium-screen) {
-    padding-right: 5%;
-  }
-}
-
-.usa-input-grid:last-of-type {
-  @include media($medium-screen) {
-    padding-right: 0;
-  }
-}
-
-.usa-input-grid-small {
-  @include media($medium-screen) {
-    float: left;
-    width: 35%;
-  }
-
-  input {
-    margin-bottom: 3rem;
-  }
-
-  select {
-    margin-bottom: 3rem;
-  }
+  @include typeset(
+    $theme-font-family-forms,
+    $theme-font-size-base,
+    $theme-line-height-input
+  );
 }
 
-.usa-input-grid-medium {
-  @include media($medium-screen) {
-    float: left;
-    width: 65%;
+.usa-form {
+  @include at-media('mobile-lg') {
+    max-width: units('mobile');
   }
 
-  input {
-    margin-bottom: 3rem;
+  .usa-input,
+  .usa-range,
+  .usa-select,
+  .usa-textarea {
+    // max width not needed since .usa-form has one of its own
+    max-width: none;
   }
 
-  select {
-    margin-bottom: 3rem;
+  .usa-input-tiny {
+    max-width: units(8);
   }
-}
 
-.usa-input-grid-large {
-  @include media($medium-screen) {
-    float: left;
-    width: 100%;
+  .usa-input-medium {
+    max-width: units(15);
   }
 
-  input {
-    margin-bottom: 3rem;
-  }
+  .usa-button {
+    margin-top: units(1);
 
-  select {
-    margin-bottom: 3rem;
+    @include at-media('mobile-lg') {
+      margin-top: units(2);
```

Comment:
`units(3)` to be the same as between input and the label?

Reply:
Done